From 7283b680dd9240f1988abe5df06397e5b94fb759 Mon Sep 17 00:00:00 2001
From: Saurav Kashyap <saurav.kashyap@qlogic.com>
Date: Mon, 25 Mar 2013 02:21:39 -0400
Subject: [PATCH 372/858] [SCSI] qla2xxx: Remove debug code that msleeps for
 random duration.

Signed-off-by: Giridhar Malavali <giridhar.malavali@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
(cherry picked from commit 4a9fa41bd404b8a572e4af7d2221e423bb729e52)

Signed-off-by: Clark Williams <williams@redhat.com>
Signed-off-by: John Kacur <jkacur@redhat.com>
---
 drivers/scsi/qla2xxx/qla_attr.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/scsi/qla2xxx/qla_attr.c b/drivers/scsi/qla2xxx/qla_attr.c
index 1d82eef..b3db9dc 100644
--- a/drivers/scsi/qla2xxx/qla_attr.c
+++ b/drivers/scsi/qla2xxx/qla_attr.c
@@ -1938,11 +1938,6 @@ qla24xx_vport_delete(struct fc_vport *fc_vport)
       "Timer for the VP[%d] has stopped\n", vha->vp_idx);
  }
 
- /* No pending activities shall be there on the vha now */
- if (ql2xextended_error_logging & ql_dbg_user)
-  msleep(random32()%10);  /* Just to see if something falls on
-     * the net we have placed below */
-
  BUG_ON(atomic_read(&vha->vref_count));
 
  qla2x00_free_fcports(vha);
-- 
1.8.3.1